lib/Jsrt/ChakraDebug.h

```diff
@@ -22,6 +22,14 @@
 
 #include "ChakraCommon.h"
 
+#if !defined(_WIN32) || !defined(_MSC_VER)
```

Comment:
I don't know what the best is, but don't want to deviate from existing usage. If we are to fix, fix them all. P.S. This entire piece might go away when TTD resolves the types issue.